target/linux/bcm53xx/base-files/etc/board.d/01_leds

```diff
@@ -5,6 +5,13 @@
 board_config_update
 
 case "$(board_name)" in
+linksys,panamera)
+	ucidef_set_led_usbport "usb2" "USB 2.0" "bcm53xx:green:usb2" "usb1-port2" "usb2-port2"
+	ucidef_set_led_usbport "usb3" "USB 3.0" "bcm53xx:green:usb3" "usb1-port1" "usb2-port1" "usb4-port1"
```

Comment:
You should not need `ucidef_set_led_usbport` if you describe LEDs precisely in the DT. See how it's done for other `BCM5301X` devices.

Reply:
Sure thing, I have those already defined in DTS, so will drop those. Thanks.